modules/ROOT/pages/storage.adoc

```diff
@@ -233,6 +233,29 @@ storage:
       label: root
 ----
 
+== Adding swap
+
+This example creates a swap partition spanning all of the `sdb` device, creates a swap area on it, and creates a systemd swap unit so the swap area is enabled on boot.
+
+.Configuring a swap partition on a second disk
+[source,yaml]
+----
+variant: fcos
+version: 1.4.0-experimental
```

Comment:
Hmm but should we really be documenting using the experimental fcc version?

Reply:
Hmm, it's a good point.  We have done it once before, in #172.  I'm okay with adding a `NOTE` or with deferring the PR outright until whenever 1.4.0 stabilizes.  Thoughts?

Reply:
IMO it's situational, for something like LUKS I think getting the documentation in early (with maybe an issue being created to track the dropping of `-experimental` once the spec lands). I'd also be fine if we just want to defer until 1.4.0 stabilizes.

Reply:
No strong opinion either way myself.

Reply:
I've added a NOTE, on the theory that it's better to document experimental FCOS functionality with caveats than not document it at all.